kubectl-fdb/cmd/remove_process_group_test.go

```diff
@@ -178,15 +178,15 @@ var _ = Describe("[plugin] remove process groups command", func() {
 					podNames          []string
 					clusterNameFilter string // if used, then cross-cluster will not work
 					clusterLabel      string
-					RemoveAllFailed   bool
```

Comment:
Was that option always false?

Reply:
yeah it wasn't looked at or used in those tests